public.Dockerfile

```diff
@@ -1,6 +1,18 @@
+FROM node:8 as node-stage
+
+COPY . /app
+
+WORKDIR /app/amundsen_application/static
+
+RUN npm install
+RUN npm run build
```

Comment:
Could you remove line 7-8 and see if it works?
This should be done via setup.py
https://github.com/lyft/amundsenfrontendlibrary/blob/master/setup.py#L24

Reply:
When removing the npm install and npm run build, it no longer works. But I see what you mean. The problem is that witht he multi-stage build node/npm is no longer available in the 2nd stage when setup.py is running. What I could do is go back to a one stage docker build and just installing node before setup.py is executed. I had that version before. The problem with that is just that it increases the image from 966MB to 1.36GB. What do you think?

Reply:
I wrote a long winded reply not worth posting along the same lines as what you just wrote @gschmutz

Another option worth considering is as a first simple step to quickly get to distributing a working container is simply not having it as a source built container but just a runtime. This can just leverage the existing pypi release of the frontend library. 

WDYT?

Reply:
Cool. Let's start with this PR as it fixes the problem that currently facing. Thanks @gschmutz !